lib/puppet/provider/nssdb/certutil.rb

```diff
@@ -0,0 +1,44 @@
+Puppet::Type.type(:nssdb).provide(:certutil) do
+  commands :certutil => 'certutil'
+
+  def create
+    generate_nssdb
+  end
+
+  def destroy
+    delete_nssdb
+  end
+
+  def exists?
+    files = resource[:nssdb_files]
+    files = [files] unless files.is_a?(Array)
+
+    files.each do |file|
+      return False unless File.exist?(file)
+    end
```

Comment:
```suggestion
    files.all? { |file| File.exist?(file) }
```

You can use the `all?` check to see if every file matches some condition. If any one of them fails then it returns false.